src/app/content/components/CallToActionPopup/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { loginLink as loginLinkSelector, signupLink as signupLinkSelector } from '../../../auth/selectors';
+import Button from '../../../components/Button';
+import htmlMessage from '../../../components/htmlMessage';
+import { closeCallToActionPopup } from '../../actions';
+import { showCTAPopup } from '../../selectors';
+import bottomLayer from './assets/bottom-layer.svg';
+import ctaGraphic from './assets/desktop-mobile-graphic.svg';
+import middleLayer from './assets/middle-layer.svg';
+import topLayer from './assets/top-layer.svg';
+import {
+  CTABackground,
+  CTABottomLayer,
+  CTAButtons,
+  CTACloseButton,
+  CTAContent,
+  CTAGraphic,
+  CTAHeading,
+  CTAMiddleLayer,
+  CTAParagraph,
+  CTAText,
+  CTATextLink,
+  CTATopLayer,
+  CTAWrapper,
+} from './styles';
+
+// tslint:disable-next-line: variable-name
+const CallToActionPopup = () => {
+  const dispatch = useDispatch();
+  const showPopup = useSelector(showCTAPopup);
+  const loginLink = useSelector(loginLinkSelector);
+  const signupLink = useSelector(signupLinkSelector);
+
+  if (!showPopup) { return null; }
+
+  const closePopup = () => {
+    dispatch(closeCallToActionPopup());
+  };
+
+  return <CTAWrapper data-analytics-region='CTA popup'>
+    <CTAContent>
+      <CTAText>
+        <CTAHeading>
+          <FormattedMessage id='i18n:cta:heading'>
+            {(msg: Element | string) => msg}
+          </FormattedMessage>
+        </CTAHeading>
+        <CtaPara />
+        <CTAButtons>
+          <FormattedMessage id='i18n:cta:button:signup'>
+            {(msg: Element | string) => <Button
+              data-testid='sign-up'
+              data-analytics-label='sign-up'
+              size='large'
+              variant='primary'
+              // eslint-disable-next-line
+              component={<a href={signupLink}/>}
+            >{msg}</Button>}
+          </FormattedMessage>
+          <CTATextLink>
+            <FormattedMessage id='i18n:cta:button:separator'>
+              {(msg: Element | string) => msg}
+            </FormattedMessage>
+            <FormattedMessage id='i18n:cta:button:login'>
+              {(msg: Element | string) => <a
+                data-testid='log-in'
+                data-analytics-label='log-in'
+                href={loginLink}
+              >{msg}</a>}
+            </FormattedMessage>
+          </CTATextLink>
+        </CTAButtons>
+      </CTAText>
+      <CTAGraphic src={ctaGraphic} alt='' />
```

Comment:
i don't think you need the alt if you put like, "role=presentation" or something like that, i'm not sure the exact value. but the empty alt seems not so good

Reply:
Linter asked me to add empty alt for presentation purposes, but I see that role presentation also works and is more descriptive

Reply:
Actually liner is shouting about this :D 
```
Prefer alt="" over a presentational role. First rule of aria is to not use aria if it can be achieved via native HTML  jsx-a11y/alt-text
```

Reply:
i mean if the linter says it it must be true